packages/react-resizable-panels/src/Panel.ts

```diff
@@ -243,7 +243,7 @@ export function PanelWithForwardedRef({
 
     // CSS selectors
     "data-panel": "",
-    "data-panel-collapsible": collapsible || undefined,
+    "data-panel-collapsible": collapsible ?? undefined,
```

Comment:
This would be a backwards breaking change for styling.

The way the rule was written before was intentional. It avoids adding the data attribute _unless_ the value is true.

(`?? undefined` would be redundant.)